static/js/typeahead_helper.js

```diff
@@ -100,10 +100,24 @@ exports.sorter = function (query, objs, get_item) {
    return results.matches.concat(results.rest);
 };
 
-exports.compare_by_pms = function (user_a, user_b) {
+exports.compare_by_pms = function (user_a, user_b, selected_recipient, current_stream_persons, current_subject_persons) {
     var count_a = people.get_recipient_count(user_a);
     var count_b = people.get_recipient_count(user_b);
-
+    if(user_a.full_name == selected_recipient){
```

Comment:
You should use `===` for comparison. This avoids a common pitfall of type conversion in JavaScript.